src/IBM.Watson.Common/Common.cs

```diff
@@ -27,7 +27,7 @@ public partial class Common
         /// <summary>
         /// The SDK version.
         /// </summary>
-        public const string Version = "watson-apis-dotnet-standard-sdk-4.0.0";
```

Comment:
It seems like this and the version in the `Common.csproj` could also be taken care of with bumpversion, but not something that needs to be done now.

Reply:
Yeah unfortunately the common version cannot be included in bumpversion. Common and core needs to be available on Nuget before CI will pass so I deployed them both already. I probably should break common out into its own repo eventually